packages/core/upload/admin/src/hooks/useAssets.js

```diff
@@ -0,0 +1,50 @@
+import { useEffect } from 'react';
+import { useQuery } from 'react-query';
+import { useNotifyAT } from '@strapi/parts/LiveRegions';
+import { useQuery as useURLQuery, useNotification } from '@strapi/helper-plugin';
```

Comment:
Here you can use the `useQueryParams` instead, it will be easier later to handle the filters and the search.